package com.somu.try_catch_finally_block;

public class NestedTryCatchFinally {
	public static void main(String[] args) {
		
		try {
//			RISKY CODE
			System.out.println("outer try block");
			try {
				System.out.println("inner try block");
				System.out.println(10/0);
			}
			catch (ArithmeticException e) {
				System.out.println("inner catch block");
			}
			
			
		} catch (Exception e) {
			System.out.println("outer catch block");
		}
		finally {
			System.out.println("outer finally block");
		}
	}
}
